src/RouteCollector.php

```diff
@@ -32,10 +32,12 @@ public function __construct(RouteParser $routeParser, DataGenerator $dataGenerat
      * The syntax used in the $route string depends on the used route parser.
      *
      * @param string|string[] $httpMethod
-     * @param string $route
-     * @param mixed  $handler
+     * @param string          $route
+     * @param mixed           $handler
+     *
+     * @return \FastRoute\RouteCollector
      */
-    public function addRoute($httpMethod, string $route, $handler):void
+    public function addRoute($httpMethod, string $route, $handler):RouteCollector
```

Comment:
Unrelated style note: I don't think this is standardized anywhere, but I believe `): RouteCollector` would be the more common formatting.